### DISCUSSION TITLE
Leaf node problem

### DISCUSSION THREAD

**Discussion by hunterljc** [nats-io/nats-server#4572] [Q&A]
Assuming that another cluster A has enabled leaf node connections, can leaf node cluster B restrict the flow of certain topics to cluster A. Some topics do not flow to cluster A and are only transmitted in leaf node cluster B.

**Comment by derekcollison** [comment]
Yes, the leafnode connection can utilize a user that has permissions that restrict what is allowed to flow over the connection.
